server.js

```diff
@@ -1,16 +1,48 @@
-const express = require('express');
+const express = require("express");
 const app = express();
+const morgan = require("morgan");
+const cors = require("cors");
 
-app.get('/getme', (req, res) => {
-  res.send(fruits)
+app.use(cors());
+app.use(morgan("dev"));
+app.use(express.json());
+
+app.get("/getme", (req, res) => {
+  res.send(fruits);
+});
+
+app.post("/postme", (req, res) => {
+  res.status(200).send(req.body);
+});
+
+app.get("/fuckup/saw-it-coming", (req, res, next) => {
+  try {
+    test;
+  } catch (err) {
+    next(err);
+  }
 });
 
-app.post('/postme', (req, res) => {
-  res.send({})
-})
+app.get("/fuckup/surprise", (req, res) => {
+  throw new Error("Error thrown");
+});
+
+app.use((req, res, next) => {
```

Comment:
Very nicely prepared the error in this route. Creating an error object and setting the status is actually a even better practice that constructing an own error object. 
Well done...